styles.css

```diff
@@ -315,29 +352,48 @@ body {
   margin-bottom: 8px;
 }
 
-.social-background {
-  background-image: url(/images/form.png);
-  background-position: bottom right;
-  background-repeat: no-repeat;
-}
-
-.primary-btn {
-  background-color: #36b37f;
-  border-radius: 4px;
-  padding: 12px;
-  align-self: center;
-  margin-top: 20px;
-  margin-bottom: 24px;
-  color: #fff;
-}
-
-.primary-btn:hover {
-  cursor: pointer;
-  background: #36b37f;
-  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
-}
-
-.primary-btn:active {
-  background-color: #008552;
-  transform: translateY(2px);
+@media only screen and (min-width: 769px) {
+  .headline {
+    background-image: url("/images/desktop-background.png");
+    background-position: top center;
+  }
+
+  .flex-responsive-wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+  }
+
+  footer {
+    background-image:
+      url(/images/bottom-left.png),
+      url(/images/bottom-right.png);
+    background-position:
+      bottom left,
+      bottom right;
+    background-repeat:
+      no-repeat,
+      no-repeat;
+    background-size: min(50%, 21rem), min(20%, 7rem);
+  }
+
+  .form-btn {
+    width: fit-content;
+  }
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] Kindly use `min-width` instead of `max-width`, currently this is affecting your background image from being displayed on desktop